src/main/java/com/ecore/roles/model/Membership.java

```diff
@@ -0,0 +1,39 @@
+package com.ecore.roles.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@Entity
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = {"team_id", "user_id"}))
```

Comment:
Add the database table name hardcoded in the Table annotation, in order to don't get a code refactoring in the class name leading to a table name mismatch in the database.